src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -592,6 +591,8 @@ export default class FilingHistoryList extends Mixins(
         item.isTypeStaff = true
         item.notationOrOrder = filing.data.order?.orderDetails // should not be falsy
         item.planOfArrangement = filing.data.order?.effectOfOrder ? 'Pursuant to a Plan of Arrangement' : ''
+        item.putBackOnOrAdminDissolution = filing.name === 'putBackOn' ||
+        filing.displayName === 'Administrative Dissolution'
```

Comment:
String codes should not be used. At a minimum, you should use enums here.

I recommend you use the mixin methods you created a few weeks back (`isTypePutBackOn()`, etc).